### PR TITLE
FLCN-138

### PR DESCRIPTION
**Pull request by MaxWardle**
Updated styling to match bc gov

### CODE REVIEW

File: src/app/registration/success/success.component.html [hunk 1]
```diff
@@ -27,12 +27,10 @@ <h1>Success, your order has been completed!</h1>
...
+        <p class="mb-0">
             We hope you enjoy your visit
         </p>
     </div>
 </div>
-<div class="text-right">
-    <button type="button" class="btn btn-primary btn-lg" (click)="navigate()">Home</button>
-</div>
\ No newline at end of file
+<button type="button" class="float-right BC-Gov-PrimaryButton" (click)="navigate()">Home</button>
```
Comment (cameronpettit) [inline src/app/registration/success/success.component.html:36]: spacing feels a bit off with the back arrow icon

File: src/assets/base/base.scss [hunk 2]
```diff
@@ -1,18 +1,248 @@
...
+        font-size: 28px;
+        margin-bottom: 0px;
+    }
+}
+
+hr {
+    border: none;
+}
+
+@media (min-width: 576px) {
```
Comment (cameronpettit) [inline src/assets/base/base.scss:232]: consolidate entries for @media of same min-width

**Review by cameronpettit — CHANGES_REQUESTED** [review]

File: src/app/pass-lookup/pass-lookup-form/pass-lookup-form.component.html [hunk 3]
```diff
@@ -9,5 +9,5 @@
     </div>
 </form>
 <div class="d-flex justify-content-end">
-    <button class="btn btn-primary" [disabled]="lookupForm.invalid" (click)="onSubmit()">Send Email</button>
+    <button class="BC-Gov-PrimaryButton" [disabled]="lookupForm.invalid" (click)="onSubmit()">Send Email</button>
```
[the remark this answers is not in the document]
Comment (MaxWardle) [reply src/app/pass-lookup/pass-lookup-form/pass-lookup-form.component.html:12]: added disabled button bcgov styling

File: src/assets/base/base.scss [hunk 4]
```diff
@@ -1,18 +1,248 @@
...
+    background-color: #003366;
+    border: none;
+    border-radius: 4px;
+    color: white;
+    padding: 12px 32px;
+    text-align: center;
+    text-decoration: none;
+    display: block;
+    font-size: 18px;
+    font-family: ‘BCSans’, ‘Noto Sans’, Verdana, Arial, sans-serif;
```
[the remark this answers is not in the document]
Comment (MaxWardle) [reply src/assets/base/base.scss:39]: added bc sans properly

File: src/app/header/header.component.html [hunk 5]
```diff
@@ -4,8 +4,21 @@
...
+<nav class="navigation-main" id="navbar">
+    <div class="container">
+      <ul>
+        <li><a href="sample.html" class="active">Visiting Parks</a></li>
+        <li><a href=".">Support Us</a></li>
+        <li><a href=".">Volunteers</a></li>
+        <li><a href=".">Conversation</a></li>
+        <li><a href=".">Planning</a></li>
+        <li><a href=".">Bc Parks</a></li>
+        <li><a href=".">Resources</a></li>
```
[the remark this answers is not in the document]
Comment (MaxWardle) [reply src/app/header/header.component.html]: removed links for now, left the navbar for future use and to match styling

**Review by mtCarto — APPROVED** [review]

**Comment by marklise** [comment]
Just a drive-by here, we should be using padding/margin etc for bootstrap classes instead of hardcoding those values in to custom classes.

File: src/assets/base/base.scss (hunk 4 above)
[the remark this answers is not in the document]
Comment (cameronpettit) [reply src/assets/base/base.scss:39]: Site still does not display BC Sans font

File: src/assets/base/base.scss [hunk 6]
```diff
@@ -1,18 +1,257 @@
...
+
+.main-content {
+    background-color: white;
+    height: 100%;
+    margin-bottom: 120px;
+}
+
+// buttons
+
+.BC-Gov-PrimaryButton {
```
Comment (cameronpettit) [inline src/assets/base/base.scss:29]: button styling classes should overrule bootstrap `btn-*` classes - `btn-primary`, `btn-secondary`, etc as per mark's comment (typ)

File: src/assets/base/base.scss [hunk 7]
```diff
@@ -1,18 +1,257 @@
...
+    }
+}
+
+@media (min-width: 576px) {
+    .main-content {
+        margin-bottom: 240px;
+    }
+
+    .header-primary {
+        margin-top: 60px;
```
Comment (cameronpettit) [inline src/assets/base/base.scss:231]: hard-coding spacing values may cause issues with adaptive display - use bootstrap spacing classes in angular template files instead. (typ)

**Review by cameronpettit — CHANGES_REQUESTED** [review]

**Comment by MaxWardle** [comment]
Closed because most of this has been addressed in other PRs since this was opened. Also some styling choices made here have been moved away from.

**Pull request closed without merging**